ophyd/ophydobj.py

```diff
@@ -61,8 +72,15 @@ def root(self):
 
     @property
     def report(self):
+        '''A report on the object.'''
         return {}
 
+    @property
+    def event_types(self):
+        '''Events that can be subscribed to via `obj.subscribe`
+        '''
+        return self._pub_event_types
```

Comment:
This happens to be returning a mutable list...

Also, do you have to add on `event_types` to the reserved keys now?